### PR TITLE
Add exception occurred within the transaction integration test case

### PR DESCRIPTION
**Pull request by FlyingZC**
Changes proposed in this pull request:
- Add exception occurred within the transaction integration test case

### CODE REVIEW

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    @Override
+    @SneakyThrows(SQLException.class)
+    protected void executeTest() {
+        Connection conn = null;
+        try {
+            conn = getDataSource().getConnection();
+            conn.setAutoCommit(false);
+            assertAccountRowCount(conn, 0);
+            executeWithLog(conn, "insert into account(id, balance, transaction_id) values(1, 1, 1);");
+            // Simulate execution exception in transaction
```
Comment (strongduanmu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java:54]: Please remove this useless doc.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            } catch (final SQLException ignored) {
+                
+            }
+        });
+        queryThread.start();
+        try {
+            queryThread.join();
+            log.info("ExceptionInTransactionTestCase test over.");
+        } catch (final InterruptedException ignored) {
+
```
Comment (strongduanmu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java:79]: Please remove this blank line.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java (hunk 1 above)
Comment (FlyingZC) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java]: I will correct it.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java (hunk 2 above)
Comment (FlyingZC) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/ExceptionInTransactionTestCase.java]: I will correct it.

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**